sdk/src/device_enumerator_ethernet.cpp

```diff
@@ -30,15 +30,18 @@
  * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
```

Comment:
device_enumerator_ethernet.h/cpp could be excluded by cmake when building without Network support.